otx/cli/manager/config_manager.py

```diff
@@ -257,11 +259,15 @@ def _get_task_type_from_data_format(self, data_format: str) -> str:
                 return task_key
         raise ConfigValueError(f"Can't find proper task. we are not support {data_format} format, yet.")
 
-    def auto_split_data(self, data_roots: str, task: str):
+    def auto_split_data(self, data_roots: str, task: str, train_ann_file: Optional[str] = None):
```

Comment:
Couldn't we use `ann_file` instead of `train_ann_file`? (This is simply because the first parameter uses data_roots.)

Reply:
Done. [fd4b79d](https://github.com/openvinotoolkit/training_extensions/pull/1921/commits/fd4b79d485996d8cdc6c43a902f1ddf25a21a227)